bitwarden-cli-alfred-workflow/info.plist

```diff
@@ -2258,39 +2469,39 @@ My thanks go out to Bitwarden for their awesome product and the new CLI!</string
 		<key>EF20647F-5BE3-4143-A667-40375A6C91F8</key>
 		<dict>
 			<key>xpos</key>
-			<integer>510</integer>
+			<integer>515</integer>
 			<key>ypos</key>
-			<integer>890</integer>
+			<integer>1130</integer>
 		</dict>
 		<key>FED5E784-7A96-4E70-BA0D-40E4C79B7C98</key>
 		<dict>
 			<key>xpos</key>
-			<integer>220</integer>
+			<integer>225</integer>
 			<key>ypos</key>
-			<integer>1130</integer>
+			<integer>1370</integer>
 		</dict>
 		<key>FF2D41F7-FB7E-4CCD-B82B-4F66FCC78F37</key>
 		<dict>
 			<key>xpos</key>
-			<integer>220</integer>
+			<integer>225</integer>
 			<key>ypos</key>
-			<integer>670</integer>
+			<integer>910</integer>
 		</dict>
 	</dict>
 	<key>variables</key>
 	<dict>
 		<key>STATUS_MESSAGE</key>
-		<string></string>
+		<string>Unlock successful.</string>
 		<key>STATUS_MESSAGE_DESC</key>
-		<string></string>
+		<string>User: christian.niehaves@icloud.com</string>
```

Comment:
This value and the one above should be empty.

Reply:
Cleared the variables with the new commit "Cleared variables"